### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #346: Create a contibuting.md
### Use-cases
This will make the process of contribution clear and set out some standards.

### Proposal
Creat a file that should includes

Types of contribution e.gm text changes, code changes and test changes
Good tickets to pick up (see labels)
How to fork
Create branch that named issue-ID
What checks Todo when making changes
The review process

### References
Examples
 
https://gist.github.com/PurpleBooth/b24679402957c63ec426

https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/.github/CONTRIBUTING.md

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

https://github.com/marcduiker/azure-functions-university/blob/main/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lazyplatypus**
## Changes made
* Added Contributing Guide

Closes #346 

@emsesc need your help shaping up the curriculum structure portion

### CODE REVIEW

**Comment by emsesc** [comment]
@lazyplatypus Should we add counselor bot documentation on the bit project website as well? In terms of documentation here, it would only be the file system structure right and the tables at the top of response files? (no config.yml stuff?)

**Comment by lazyplatypus** [comment]
yes let's add it into our issues

**Comment by blueboxes** [comment]
The Step 2 - Process talks about new lessons did you also want to add something about to get started picking up one of the existing issues,  and the processes for something like a small text change. 2 code reviewers etc.

Is it also worth mentioning that the step instructions can be found in the .bit folder?

**Comment by emsesc** [comment]
> The Step 2 - Process talks about new lessons did you also want to add something about to get started picking up one of the existing issues, and the processes for something like a small text change. 2 code reviewers etc.
> 
> Is it also worth mentioning that the step instructions can be found in the .bit folder?

@blueboxes We will be adding that in a section about the configuration of the bot, file structure etc.

## Curriculum Structure
- [ ] File structure
- [ ] Do's and don'ts
- [ ] The best way to write a step

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Once we've discussed the Content Request issue and agree to include the lesson, you can start by forking the [Intro to Serverless repository](https://github.com/bitprj/Intro-To-Serverless) and create a new branch for the topic you want to add
+
+Once you've created the lesson, submit a PR with the lesson, source code, and related files as mentioned in the next section. It will probably take a few iterations to get everything right, don't feel discouraged by this process ♥.
+
+### Curriculum Structure
+
+If you are contributing to create a lesson, please take into account the length and the tone of the existing lessons. We want to have a uniform experience across all of our lessons.
+
+Our curriculum consists of several parts:
```
Comment (blueboxes) [inline CONTRIBUTING.md:23]: Can I suggest 

> Students advance through the curriculum by committing code. These commits are checked by the councilor bot and then students advanced to the next section where further instructions are provided by the bot. For this to work the curriculum stages need to be created in a specific format, our curriculum consists of several parts

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+👋 Hi! Great that you want to contribute to Intro-to-Serverless! 😃
+
+We're currently focussed on creating beginner content for Azure Functions written in Javascript, and using VSCode as the code editor.
+
+## Step 1 - GitHub Issues
+
+🔎 Please check if there's an [existing issue](https://github.com/bitprj/Intro-To-Serverless/issues) which matches your idea. Perhaps you can collaborate with someone on this.
+
+💡 If you have  new idea, please [create a new __Content Request__ issue](https://github.com/bitprj/Intro-To-Serverless/issues/new) first where you can describe the topic.
+
```
Comment (blueboxes) [inline CONTRIBUTING.md:12]: A note about the best way is to help resolve an existing issue marked as good first issue.

**Review by emsesc — APPROVED** [review]

**Review by blueboxes — APPROVED** [review]

**Pull request merged**